Misc/NEWS.d/next/Library/2025-08-12-19-10-25.gh-issue-137630.n6CkjJ.rst

```diff
@@ -0,0 +1,2 @@
+The :mod:`!_interpqueues` module now uses Argument Clinic to parse arguments.
```

Comment:
We usually do not add NEWS files for conversions to Argument Clinic. Argument Clinic is an implementation detail.